changelog/unreleased/20200514104636_enable_layout_spacearoundmethodcalloperator_cop.yml

```diff
@@ -0,0 +1,5 @@
+---
+type: added
+title: Enable Layout/SpaceAroundMethodCallOperator cop
+username:
+issue:
```

Comment:
```
username:
issue:
```
@javierav Shouldn't we be consistent with the rest of the entries, and remove these two lines?

Reply:
Removed! Good catch!